consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/request/WarrantRequest.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.moj.sdt.cmc.consumers.request;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import uk.gov.moj.sdt.cmc.consumers.request.judgement.Address;
+import uk.gov.moj.sdt.cmc.consumers.request.judgement.SotSignature;
```

Comment:
As these classes are used by more than one request type, should they be moved to a common package?

Reply:
done